com.valvesoftware.Steam.metainfo.xml

```diff
@@ -73,33 +73,10 @@
     <release version="1.0.0.62" date="2020-03-19"/>
     <release version="1.0.0.61" date="2019-04-18"/>
   </releases>
-    <content_rating type="oars-1.1">
-    <content_attribute id="violence-cartoon">intense</content_attribute>
-    <content_attribute id="violence-fantasy">intense</content_attribute>
-    <content_attribute id="violence-realistic">intense</content_attribute>
-    <content_attribute id="violence-bloodshed">intense</content_attribute>
-    <content_attribute id="violence-sexual">intense</content_attribute>
-    <content_attribute id="violence-desecration">intense</content_attribute>
-    <content_attribute id="violence-slavery">intense</content_attribute>
-    <content_attribute id="violence-worship">intense</content_attribute>
-    <content_attribute id="drugs-alcohol">moderate</content_attribute>
-    <content_attribute id="drugs-narcotics">moderate</content_attribute>
-    <content_attribute id="drugs-tobacco">moderate</content_attribute>
-    <content_attribute id="sex-nudity">intense</content_attribute>
-    <content_attribute id="sex-themes">intense</content_attribute>
-    <content_attribute id="sex-homosexuality">moderate</content_attribute>
-    <content_attribute id="sex-prostitution">moderate</content_attribute>
-    <content_attribute id="sex-adultery">moderate</content_attribute>
-    <content_attribute id="sex-appearance">intense</content_attribute>
-    <content_attribute id="language-profanity">intense</content_attribute>
-    <content_attribute id="language-humor">intense</content_attribute>
-    <content_attribute id="language-discrimination">intense</content_attribute>
+  <content_rating type="oars-1.1">
     <content_attribute id="social-chat">intense</content_attribute>
-    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-info">intense</content_attribute>
     <content_attribute id="social-audio">intense</content_attribute>
-    <content_attribute id="social-location">none</content_attribute>
-    <content_attribute id="social-contacts">none</content_attribute>
```

Comment:
IIRC Steam allows sharing email, correct me if I'm wrong.

Reply:
Right, you're correct, sharing email addresses counts as intense sharing of social-contacts.

Reply:
Also possibly if Steam doesn't moderate communication
```
    <content_attribute id="social-chat">intense</content_attribute>
    <content_attribute id="social-audio">intense</content_attribute>
    <content_attribute id="social-contacts">intense</content_attribute>
```

Reply:
> Also possibly if Steam doesn't moderate communication
> 
> ```
>     <content_attribute id="social-chat">intense</content_attribute>
>     <content_attribute id="social-audio">intense</content_attribute>
>     <content_attribute id="social-contacts">intense</content_attribute>
> ```

Rating `social-chat` was tricky because chat does have automated filters which are turned on by default. But they don't catch everything and human mods usually aren't watching, so I marked it `intense` rather than `moderate`.